src/map/script.cpp

```diff
@@ -7855,6 +7856,11 @@ BUILDIN_FUNC(getitem2)
 					return SCRIPT_CMD_FAILURE;
 				}
 			}
+			int maxlen = 256;
+			char *buf = (char *)aMalloc(maxlen*sizeof(char));
+			memset(buf, 0, maxlen);
+			snprintf(buf, maxlen-1, "%llu", (unsigned long long)unique_id);
+			mapreg_setregstr(amount > 1? reference_uid(add_str("$@unique_id$"), i):add_str("$@unique_id$"), buf);
```

Comment:
```suggestion
			mapreg_setregstr(reference_uid(add_str("$@unique_id$"), i), buf);
```

Reply:
`$@` is a misuse, please use the scope variable `.@`